### ISSUE TITLE
[Moderate] Graphical object lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#53]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87906 |
| **Checkpoint** | 1.4 Distinguishable › Non-Text Contrast - Graphical Objects (1.4.11.c) |
| **Impact** | Moderate |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Dashboard |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/dashboard> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/15/26 at 11:01 am |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Parts of graphics (required to understand the content) do not have a contrast ratio of 3 to 1 against adjacent color(s).
While the gold (for Glacier-Deep-OH) doesn't show in the Deposits by Storage Option pie chart, and it has equivalent table data, the gold is just below 3 to 1. The pie chart might be helpful to visual people (who do not like tables), so it would be good to have the color dark enough to make it easy to see.
Also occurs on 03 Object, select an Object from the table, on Object Detail page, in File Summary section.

## Recommendation

Rule : 
Parts of graphics (required to understand the content) MUST have a contrast ratio of 3 to 1 against adjacent color(s). Exceptions exist.

How To Fix: 
Fix this issue by adjusting the parts of graphics (that are required to understand the content) and/or adjacent color to increase the contrast to at least 3 to 1.

Reference: 
Accessible Charts and Graphs: https://accessibility.psu.edu/images/charts/
Accessible Data Visualizations: https://accessibility.huit.harvard.edu/data-viz-charts-graphs
WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

Background: 
People who have low vision or are colorblind may have difficulty perceiving the important parts of an informative graphical object (i.e., the parts of the graphic required to understand the content) if the contrast between parts of the graphic essential for understanding the graphic is insufficient. When an graphical object has adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive the information conveyed by the object.
NOTE: The term "graphical object" applies to stand-alone icons such as a print icon (with no text), and the important parts of a more complex diagram such as each line in a graph. For simple graphics such as single-color icons the entire image is a graphical object. Images made up of multiple lines, colors and shapes will be made of multiple graphical objects, some of which are required for understanding. Not every graphical object needs to contrast with its surroundings - only those that are required for a user to understand what the graphic is conveying. This requirement does not apply to a graphic whose information is available in another form such as text - for example a printer icon with the word "print" next to it - and it does not apply to decorative images that do not convey information.

## Source Code

```
<td class="text-sm">
                
                Glacier-Deep-OH
                
              </td>
```

## Screenshots

![01_graphical contrast](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87906/01_graphical_contrast.png)
![03_graphical](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87906/03_graphical.png)
